linkerd/app/inbound/src/policy/discover.rs

```diff
@@ -109,6 +109,21 @@ fn to_policy(proto: api::Server) -> Result<ServerPolicy> {
         _ => return Err("proxy protocol missing".into()),
     };
 
+    let loopback = Authorization {
+        name: "default:localhost".into(),
+        authentication: Authentication::Unauthenticated,
+        networks: vec![
+            Network {
+                net: "127.0.0.1".parse::<IpAddr>().unwrap().into(),
+                except: vec![],
+            },
+            Network {
+                net: "::1".parse::<IpAddr>().unwrap().into(),
```

Comment:
similarly this could be 
```suggestion
                net: IpAddr::from([0, 0, 0, 0, 0, 0, 0, 1]).into(),
```